autogtp/Game.cpp

```diff
@@ -65,17 +65,23 @@ void Game::error(int errnum) {
 }
 
 bool Game::eatNewLine() {
-    char readBuffer[256];
+    char readBuffer[256] = {0};
+    qint64 readCount = 0;
     // Eat double newline from GTP protocol
     if (!waitReady()) {
         error(Game::PROCESS_DIED);
         return false;
     }
-    auto readCount = readLine(readBuffer, 256);
-    if(readCount < 0) {
-        error(Game::WRONG_GTP);
-        return false;
-    }
+    do {
+        waitForReadyRead(100);
```

Comment:
Magic delay with a random value? Why is this needed? Why this value? I won't merge this...